contracts/Lens/VenusLens.sol

```diff
@@ -58,6 +69,9 @@ contract VenusLens is ExponentialNoError {
             underlyingDecimals = EIP20Interface(vBep20.underlying()).decimals();
         }
 
+        Comptroller comptrollerInstance = Comptroller(comptrollerAddress);
```

Comment:
Why do we need two different interfaces for one contract? I propose we either extend `LensInterface` or remove it in favor of `Comptroller`. The latter would increase the compilation time, but would bring some safety since it would guarantee that the interface matches the implementation. Another option would be to add all the missing methods to `ComptrollerInterface` and import it here.

@RoseRompkxm @defistar what do you guys think?

Reply:
@kkirka @RoseRompkxm @narayanprusty 

We can add all of them to comptrollerInterface
&
get rid of LensInterface

shall i do it?

```
interface LensInterface {
    function markets(address) external view returns (bool, uint);
    function oracle() external view returns (PriceOracle);
    function getAccountLiquidity(address) external view returns (uint, uint, uint);
    function getAssetsIn(address) external view returns (VToken[] memory);
    function claimVenus(address) external;
    function venusAccrued(address) external view returns (uint);
}
```

Reply:
@defistar still, do we really need `Comptroller comptrollerInstance` here? What about we add `function venusSpeeds(address)` to `ComptrollerInterface` and remove `import "Comptroller.sol` altogether?

Reply:
```
    function venusSpeeds(address) external view returns (uint);
    function getAllMarkets() external view returns (VToken[] memory);
    function venusSupplierIndex(address, address) external view returns (uint);
    function venusInitialIndex() external view returns (uint224);
    function venusBorrowerIndex(address, address) external view returns (uint);
    function venusBorrowState(address) external view returns (uint224, uint32);
    function venusSupplyState(address) external view returns (uint224, uint32);
```

added these functions to ComptrollerInterface which were earlier refered directly from Comptroller
all these are being used by VenusLens (edited)